src/scene/selection.ts

```diff
@@ -67,3 +68,23 @@ export const getTargetElements = (
   appState.editingElement
     ? [appState.editingElement]
     : getSelectedElements(elements, appState);
+
+export const getSelectedPoint = (
+  elements: readonly NonDeletedExcalidrawElement[],
+  appState: AppState,
+) => {
+  if (
+    appState.editingLinearElement &&
+    appState.editingLinearElement.activePointIndex != null
```

Comment:
```suggestion
    appState.editingLinearElement?.activePointIndex !== null
```

Reply:
Oh right, wasn't sure I could use optional chaining!

Reply:
@lipis I used `!=` with the intent to check `activePointIndex` is not null nor undefined.

If you prefer, I can do:
```suggestion
    appState.editingLinearElement?.activePointIndex !== null &&
    appState.editingLinearElement?.activePointIndex !== undefined
```